FishBun/src/main/res/layout/album_item.xml

```diff
@@ -8,37 +8,38 @@
 
     <com.sangcomz.fishbun.util.SquareImageView
         android:id="@+id/img_album_thumb"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:layout_centerInParent="true"
+        android:layout_width="60dp"
+        android:layout_height="60dp"
         android:scaleType="centerCrop" />
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_marginLeft="10dp"
+        android:paddingLeft="10dp"
```

Comment:
Because if your margin when we select album then select animation does not look good, using padding it complete animation. Only this is important change